msf_updates/src/pose_msf/pose_sensormanager.h

```diff
@@ -170,16 +170,16 @@ class PoseSensorManager : public msf_core::MSF_SensorManagerROS<
     P.setZero();  // Error state covariance; if zero, a default initialization in msf_core is used
 
     p_vc = pose_handler_->GetPositionMeasurement();
-    q_cv = pose_handler_->GetAttitudeMeasurement();
+    q_vc = pose_handler_->GetAttitudeMeasurement();
```

Comment:
If I understand correctly, the first attitude measurement was always added inversed in the case of vicon... Ok, vicon measurements are added at a high rate so the effect was maybe never visible...